src/cli/peakdetector/PeakDetectorCLI.cpp

```diff
@@ -50,8 +51,18 @@ void PeakDetectorCLI::processOptions(int argc, char* argv[]) {
 
 		case 'a':
 			mavenParameters->alignSamplesFlag = true;
-			if (atoi(optarg) == 0) mavenParameters->alignSamplesFlag = false;
-			break;
+
+            if (atoi(optarg) == 1)
+                alignType = OBI_WARP;
+
+            else if (atoi(optarg) == 2)
+                alignType = POLY_FIT;
+
+            else
+                alignType = NO_TYPE;
+                mavenParameters->alignSamplesFlag = false;
```

Comment:
@rish9511 won't the flag always be set as false? else block needs to be within brackets

Reply:
@shubhra-agrawal 
Yes, this needs to be within brackets. 
Forgot somehow